### ISSUE TITLE
[Components and pattern standards] - Your content or feature doesn't use standard global navigation components or is missing a standard navigation component (04.02.1)

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#35493]
#### VFS team name
Public Websites

#### VFS product name
Outreach & Events Enhancements

#### Point of Contact/Reviewers (Practice area)
Mikki Northuis (@mnorthuis) - VSA IA Lead

---

### Platform Issue
Your content or feature doesn't use standard global navigation components or is missing a standard navigation component

### Issue Details
The "Home" link/segment is missing form the breadcrumb from all pages within (except the main landing page) Outreach and events experience.  

### VA.gov Experience Standard
04.02.1

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Add the "Home" link/segment to every page to align with the Breadcrumb design standards

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket. 

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Product Support board
- Add your feedback
- Add assignees: VFS PM
- Add labels:
  - VFS team label
  - VA.gov Experience Standard Issue number (CCIssueXX.YY) where XX is the category number and YY is the issue number, per the VA.gov Experience Standards
  - Practice area labels (508/accessibility, IA, content, design) per the VA.gov Experience Standards
  - 508 defect label, if applicable (508-defect-0, 508-defect-1, 508-defect-2, 508-defect-3, 508-defect-4)
  - Launch-blocking label, if applicable

**Comment by brianalloyd** [comment]
@mnorthuis would like to sync on this issue with @davidconlon in order to determine if VA.gov home breadcrumb pattern makes more sense for this element.  I'm also checking on the success of VET Centers recent VA.gov breadcrumb success to see if that can be applied here in a similar fashion.  Look forward to linking up in support.

**Comment by mnorthuis** [comment]
Sure, happy to chat.  The feedback is based on our breadcrumb standards.  We can discuss the label of the "home" link (which on Vet Centers is "VA.gov home"), but the standard does require that there is a home link.

**Comment by srsuddath** [comment]
This ticket has been accomplished by the following PRs
https://github.com/department-of-veterans-affairs/content-build/pull/958
https://github.com/department-of-veterans-affairs/vets-website/pull/20128
https://github.com/department-of-veterans-affairs/content-build/pull/956
